style.css

```diff
@@ -13,248 +9,196 @@ html {
   --cards-border: #dfe1e6;
   --about-subcategories-lan-fra-ski: #f7f7f9;
   --button-pressed: #2230d2;
+  --background-color-main: #f4f5f7;
 }
 
 * {
+  padding: 0%;
   margin: 0%;
+  box-sizing: border-box;
 }
 
-header .nav-bar {
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: var(--background-color-main);
+}
+
+header {
+  width: 100%;
+  height: 10vh;
   background-color: white;
```

Comment:
- [ ] Kindly rearrange the navigation header style for mobile view. It's not responsive at the moment. Your portfolio website should be responsive on all screen sizes, mobile, and desktop. Please Fix that 👍
- Your Design
![image](https://user-images.githubusercontent.com/50941074/160213556-bd6acd2d-3236-45eb-9e5b-56168a156715.png)

- Figma design
![image](https://user-images.githubusercontent.com/50941074/160213606-6c5a8239-20ef-4e0c-9774-d16f94f4b6b9.png)